Taskfile.yml

```diff
@@ -77,18 +77,71 @@ tasks:
     deps:
       - 8.3:docker:build
     cmds:
-      - docker compose run php composer exec phpcs -- src/
+      - docker compose run -T php composer exec phpcs -- src/
 
   lint:fix:
     desc: Lint fix the php code
     deps:
       - 8.3:docker:build
     cmds:
-      - docker compose run php composer exec phpcbf -- src/ tests/
+      - docker compose run -T php composer exec phpcbf -- src/ tests/
 
-  build:
+  dist:
     desc: Build zip dist package using Docker
     deps:
       - 8.3:docker:build
     cmds:
-      - docker compose run php ./build.sh
+      - docker compose run php ./scripts/build.sh
+
+  php-compatibility:
+    desc: Check compatibility code
+    cmds:
+      - docker compose run -T php composer exec phpcs -- -p src --standard=PHPCompatibility -s --runtime-set testVersion 5.6-8.1 --ignore=\*/vendor/\*
+
+  gh-cli:
+    internal: true
+    preconditions:
+      - sh: gh --version
+        msg: |
+          ⚠️ This task requires `gh` (Github CLI).
+          Please check out: https://github.com/cli/cli#installation
+          You'll need a personal access token to authenticate with `gh`.
+          Please check out: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#creating-a-personal-access-token-classic
+          Then, run `gh auth login` or set the GITHUB_TOKEN environment variable with your personal access token.
+
+  release:
+    deps:
+      - gh-cli
+    desc: Create a release pull request
+    cmds:
+      - gh workflow run release-pull-request.yml
+      - sleep 2
+      - cmd: echo "Release pull request created, check out https://github.com/alma/{{.REPOSITORY}}/pulls?q=is%3Aopen+is%3Apr+label%3Arelease"
```

Comment:
`{{ .REPOSITORY }}` is never set anywhere ?

Reply:
Indeed🤦
Thanks, I pushed a fix